jit/v2/JIT_v2.0.0.yaml

```diff
@@ -191,7 +253,13 @@ paths:
       operationId: omit-port-call
       summary: Omits a Port Call
       description: |
-        Provider `OMITTING` a **Port Call**
+        Provider `OMITTING` a **Port Call**. This is called whenever the provider has to omit a **Port Call**.
```

Comment:
Allows the provider to `OMIT` a **Port Call**, signaling that the Port Call is no longer going to happen.